### PR TITLE
Add disconnect

### PR DESCRIPTION
**Pull request by Lehren**
Add disconnect method and change specs to make them individually runnable.

### CODE REVIEW

File: src/administrative-sdk/connection/connection-controller.js [hunk 1]
```diff
@@ -140,7 +141,12 @@ export default class Connection {
       self._session = null;
       self.fireEvent('websocketClosed');
```
Comment (joggienl) [inline src/administrative-sdk/connection/connection-controller.js]: I noticed previous notations of 'webSocket'. Here 'websocket' is used. Maybe use webSocket for consistency?

**Comment by kvdb** [comment]
@joggienl I guess you didn't merge because you weren't in the team? You're admin now, so go right ahead.

**Comment by joggienl** [comment]
No, I saw pending request on your side. I was already admin on this team if I'm correct ;-)

**Comment by kvdb** [comment]
I assume Loek wanted either of us to review.

**Pull request merged**